packages/fxa-admin-server/src/newsletters/basket.service.ts

```diff
@@ -0,0 +1,92 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import fetch from 'node-fetch';
+import { AppConfig } from '../config';
+
+/**
+ * A service for basket and newsletter APIs.
+ */
+@Injectable()
+export class BasketService {
```

Comment:
The more I see the NestJS service pattern the more I like it. Keeps things much simpler. 

Not for this PR but it would be nice to expose some of these options in our settings page and let a user unsubscribe there was well, verses having an extra step of going into Basket.

Reply:
Totally agreed. The DI is pretty awesome. I also really like the fact that it's easy to shift and lift these to fxa-shared as needed, so the service can be reused across APIs.